templates/emails/reviewers_group_invitation.rst

```diff
@@ -0,0 +1,17 @@
+Hello,
+We hope you are in good health and doing well.
+This email is in regard to the PyCon India 2020 review team. PyCon India is going virtual this year considering the current situations (https://twitter.com/pyconindia/status/1253732245635952643). We, the conference organizers, are taking this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and more accessible to our esteemed attendees from all over the world in more than one way.
+
+Hence, we would like to put push towards bringing up some quality talks to be presented this  Pycon 2020. We are creating a team to review the talk proposals that should be presented in Pycon. Since this is a peer-review process and driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience in Python and/or with Python community will be a core part of this team we intend to build going ahead. 
```

Comment:
```suggestion
Hence, we would like to put push towards bringing up some quality talks to be presented this PyCon India 2020. We are creating a team to review the talk proposals that should be presented in PyCon India. Since this is a peer-review process and driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience in Python and/or with Python community will be a core part of this team we intend to build going ahead. 
```